draft-rfcs/RFC-github-access-policy.md

```diff
@@ -0,0 +1,54 @@
+# GitHub Access Policy RFC
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a policy for managing permissions to the [Rust Lang GitHub Organization](https://www.github.com/rust-lang) and repositories within this organization.
```

Comment:
maybe we should also add a note (somewhere) that this  RFC was written as part of the governance WG, in collaboration with infra folks, and give links to the issue(s) where things were discussed